deploy/helm-chart/kube-mail/templates/deployment.yaml

```diff
@@ -78,7 +78,7 @@ spec:
               value: {{ include "chart.redis.port" . | quote }}
             {{- end }}
             - name: DEBUG
-              value: "*"
+              value: "*,-express:router"
```

Comment:
Wouldn't it be best to expose this as a helm value?
I would assume no everyone wants to run this with full (or a little less full) debug output.

Reply:
Good suggestion. Added in 008ec9d.